app/src/charts/HorizontalGroupedBarChart.tsx

```diff
@@ -137,7 +137,14 @@ function getSpec(
 
 function HorizontalGroupedBarChart(props: { data: Row[]; measure: string }) {
   return (
-    <Vega spec={getSpec(props.data, "state_name", "race", props.measure)} />
+    <Vega
+      spec={getSpec(
+        props.data,
+        "state_name",
+        "race_and_ethnicity",
```

Comment:
We could type this as BreakdownCol in getSpec for additional safety

Reply:
Good call! I wanted to do this across the codebase but also thought this was starting to get out of scope haha